Fix incorrect red marker showing for both correct and wrong answers

AliHassan245 opened the pull request:
Fix #19

quozl commented:
Please don't raise an issue for a change you already have plans to make, as it increases notifications and doesn't concentrate the information in the right place.

The right place is the commit messages, and yours are very weak, because they don't follow our guidance and don't contain enough information.

The information you placed in the issue is the kind of thing that should go into the commit messages,

Please rewrite your commit messages and force push, thanks.

chimosky commented on bounce.py:
```diff
@@ -312,7 +312,10 @@ def set_background(self, name):
         ''' Pause play when visibility changes '''
         if self._step_sid is not None:
-            GLib.source_remove(self._step_sid)
+            try:
+              GLib.source_remove(self._step_sid)
+            except Exception as e:
+              pass
```
Why is there a try clause in an if statement?

chimosky replied:
Something you added for development in your local environment shouldn't be part of your changes here.

Commit:
Display visual feedback for correct and incorrect answers

Position wedge indicator off-screen by default at game start

Display visual indicator based on result passed from test method (green for correct, red for incorrect)

quozl commented:
Reviewed commit message and changes.  I don't see how you position off-screen, can you please clarify?

Code has been copy pasted.  It should be made into a function or method.
